opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -1959,10 +1962,181 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
     }
 
     const std::vector<int>& EclipseGrid::getActiveMap() const {
-
         return m_active_to_global;
     }
 
+    void EclipseGrid::assertLabelLGR(const std::string& label) const{
+        if (std::find(all_lgr_labels.begin(), all_lgr_labels.end(), label) == all_lgr_labels.end()){
+            throw std::invalid_argument("LGR label not found");
+        }            
+    }
+
+
+    void EclipseGrid::assertIndexLGR(size_t localIndex) const {
+     if (std::find(lgr_active_index.begin(), lgr_active_index.end(), localIndex) != lgr_active_index.end()) {
+        throw std::invalid_argument("input provided is an LGR refined cell");
```

Comment:
I might need first to know how lgr_active_index is populated and what localIndex represents here, but in principle, I do not get the throw-message. Is there a "not" missing? (input provided is *not* an LGR refined cell)